serialization/numpy_npy.ksy

```diff
@@ -0,0 +1,157 @@
+meta:
+  id: numpy_npy
+  title: A Simple File Format for NumPy Arrays
+  application: numpy
+  file-extension: npy
+  license: Unlicense
+  endian: le
+  ks-opaque-types: true
+  xref:
+    wikidata: Q197520
+doc: |
+  Serialization format used by numpy. An antipattern, because:
+    * uses dicts in python syntax:
+      * in order to parse the array one has to have a parser for python language syntax. This cripples interoperability: noone wants to create a parser for python syntax to just parse a numpy array.
+      * it is tempting to `exec` these dicts, which could have been a security issue.
+      * they had to use `ast.parse` and manual walking of the tree in order to mitigate that. Fortunately now it is built into python as `ast.literal_eval`.
+    The schema of the dicts is fixed, the values in the dicts are integers, the better solution would be to put these numbers as standardized variable-length integers.
```

Comment:
>The reason for this is explained in the numpy docs - it makes it easier to reverse-engineer the format from scratch in the future, because a structured text representation is easier for humans to understand than packed integer fields. It also makes it easier to add more fields to the header in the future.

Yes, it is. But they can have used json for that.